templates/base-service-template/package.json

```diff
@@ -15,12 +26,44 @@
   "author": "",
   "license": "",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "2.9.0",
-    "@typescript-eslint/parser": "2.9.0",
-    "eslint": "6.7.1",
-    "eslint-config-airbnb-base": "14.0.0",
-    "eslint-plugin-import": "2.18.2",
-    "husky": "3.1.0"
+    "husky": "3.1.0",
+    "@nestjs/cli": "^6.9.0",
```

Comment:
please freeze versions of all dependencies

Reply:
Done